.devcontainer/Dockerfile

```diff
@@ -0,0 +1,64 @@
+#-------------------------------------------------------------------------------------------------------------
```

Comment:
What pat does Docker play in accomplishing this?

Reply:
Nothing. This is related to this PR which I based this branch on:

https://github.com/Azure/azure-sdk-for-net/pull/13827

I did this because I was doing the experimentation with this change on my Linux machine.